doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -123,9 +123,9 @@ The following table indicates the software maturity levels of the support for th
           Play and pause emulated by disabling and enabling stream, respectively.
         - The following limitations apply:
 
-          * Basic Audio Profile (BAP) broadcast, one BIG with two BIS streams.
+          * One BIG with two BIS streams.
           * Audio input: USB or I2S (Line in or using Pulse Density Modulation).
-          * Configuration: 16 bit, several bit rates ranging from 24 kbps to 160 kbps.
+          * Configuration: 16 or 32 bit, several bit rates ranging from 32 kbps to 124 kbps.
```

Comment:
Can you double check CI coverage on this?

Reply:
Goes for all 32 bit mentions.

Reply:
Not sure 32 bit should be mentioned until we have discussed. Some modules do not support 32 bit.

Reply:
Remove 32 bit for now